src/coverlet.collector/version.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
+    "version": "1.0.0-build.{height}",
```

Comment:
@onovotny did it but no luck
![image](https://user-images.githubusercontent.com/7894084/58367332-50814780-7ede-11e9-8632-e8ae29858acf.png)